onnxruntime/core/providers/cuda/math/unary_elementwise_ops.cc

```diff
@@ -124,12 +124,18 @@ Status UnaryElementwise::Prepare(OpKernelContext* context, UnaryElementwisePrepa
   UNARY_OP_TYPED(name, ver, float)     \
   UNARY_OP_TYPED(name, ver, double)
 
+#define UNARY_OP_HFDB(name, ver)       \
```

Comment:
'B' is taken by uint8_t type. In other EP (onnxruntime/core/providers/cann/math/unary_elementwise_ops.cc), 'X' is used for BFloat16. Can we follow the same prefix?